### DISCUSSION TITLE
FNF B-Sides Mod

### DISCUSSION THREAD

**Discussion by DzGithub2805** [luckydog7/Funkin-android#1406] [mods]
Download link: https://drive.google.com/file/d/1G4hDs7jNAIk1OXXaVpF13Km6Ln0AvluJ/view?usp=drivesdk

Preview: 
![Screenshot_20210623-113254](https://user-images.githubusercontent.com/83911327/123036891-f6824180-d417-11eb-8401-3aa59fe57400.jpg)
![Screenshot_20210623-113307](https://user-images.githubusercontent.com/83911327/123036903-f97d3200-d417-11eb-8e95-d0894fa37030.jpg)
![Screenshot_20210623-113357](https://user-images.githubusercontent.com/83911327/123036907-fc782280-d417-11eb-90bb-cc46aa280c1e.jpg)
![Screenshot_20210623-113430](https://user-images.githubusercontent.com/83911327/123036913-000ba980-d418-11eb-86c8-9dca80da321c.jpg)
![Screenshot_20210623-113515](https://user-images.githubusercontent.com/83911327/123036920-026e0380-d418-11eb-99bc-44cf9d79ca14.jpg)
![Screenshot_20210623-113550](https://user-images.githubusercontent.com/83911327/123036925-06018a80-d418-11eb-9036-20516cf7d270.jpg)
All weeks and also tiky, this is kade engine demo 3 by zack

(Monster is remixed too) 🤣

**Comment by DiosDonutSlave** [comment]
Doesn't work, cause touch controls broken

**Reply by DzGithub2805** [reply]
I'm worked, maybe it just ur phone
